simulation_parameters/Constants.cs

```diff
@@ -412,6 +412,11 @@ public static class Constants
     public const float TOOLTIP_DEFAULT_DELAY = 1.0f;
 
     public const float TOOLTIP_FADE_SPEED = 0.1f;
+    
+    /// <summary>
+    ///   Increment to AI membrane microbes to make hovering over small cells easier
```

Comment:
I still find this comment confusing / it doesn't correctly explain what this variable is for.
Also CI is still unhappy about the trailing whitespace.